other-products/subbridge/supported-assets.md

```diff
@@ -1,13 +1,11 @@
 # Supported Assets
 
-Currently, we have supported the following assets with the corresponding transfer routes. The table below shows which chains have been supported for each asset. For example, we can transfer ZLK between parachains and Moonriver EVM, but we cannot transfer it between parachains and Ethereum.
+An asset id of type _uint32_ is allocated for each registered asset. Each asset except PHA on Khala/Phala will have an unique asset id. You can use [pallet-assets](https://github.com/paritytech/substrate/tree/master/frame/assets) to transfer asset to any accounts within Khala and Phala Network.
 
-An asset id of type _uint32_ is allocated for each registered asset. Each asset except PHA on Khala/Phala will have an unique asset id. You can use [pallet-assets](https://github.com/paritytech/substrate/tree/master/frame/assets) to transfer asset to any accounts within Khala/Phala Network.
+* Assets on the Khala network
 
-* Assets on Khala network
+Foreign assets are managed by pallet-assets, supported assets can be found at [here](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fkhala-api.phala.network%2Fws#/assets). On the Khala network, PHA can be transferred between parachain and Ethereum, and ZLK can be transferred to Moonriver and vice versa.
 
-<table><thead><tr><th>Name</th><th width="229">Location</th><th>Asset ID</th><th>Parachains</th><th>Ethereum</th><th>Moonriver EVM</th></tr></thead><tbody><tr><td>PHA</td><td>(1, X1(Parachain(2004)))</td><td>null</td><td>✅</td><td>✅</td><td>❌</td></tr><tr><td>KSM</td><td>(1, Here)</td><td>0</td><td>✅</td><td>❌</td><td>❌</td></tr><tr><td>KAR</td><td>(1, X2(Parachain(2000), GeneralKey(0x0080)))</td><td>1</td><td>✅</td><td>❌</td><td>❌</td></tr><tr><td>BNC</td><td>(1, X2(Parachain(2001), GeneralKey(0x0001)))</td><td>2</td><td>✅</td><td>❌</td><td>❌</td></tr><tr><td>ZLK</td><td>(1, X2(Parachain(2001), GeneralKey(0x0207)))</td><td>3</td><td>✅</td><td>❌</td><td>✅</td></tr><tr><td>AUSD</td><td>(1, X2(Parachain(2000), GeneralKey(0x0081)))</td><td>4</td><td>✅</td><td>❌</td><td>❌</td></tr><tr><td>MOVR</td><td>(1, X2(Parachain(2023), PalletInstance(10)))</td><td>6</td><td>✅</td><td>❌</td><td>❌</td></tr><tr><td>HKO</td><td>(1, X2(Parachain(2085), GeneralKey(“HKO”)))</td><td>7</td><td>✅</td><td>❌</td><td>❌</td></tr><tr><td>KMA</td><td>(1, X1(Parachain(2084)</td><td>8</td><td>✅</td><td>❌</td><td>❌</td></tr><tr><td>BSX</td><td>(1, X2(Parachain(2090), GeneralIndex(0)))</td><td>9</td><td>✅</td><td>❌</td><td>❌</td></tr><tr><td>TUR</td><td>(1, X1(Parachain(2114)</td><td>10</td><td>✅</td><td>❌</td><td>❌</td></tr><tr><td>CRAB</td><td>(1, X2(Parachain(2105), PalletInstance(5)))</td><td>11</td><td>✅</td><td>❌</td><td>❌</td></tr></tbody></table>
+* Assets on the Phala network
```

Comment:
Is it worth adding the assets in a table so users don't need to go to Polkadot.js.org?

Reply:
I personally think polkadot.js.org is straightforward, here I'd like to only mention assets that enabled solo EVM chain transferring. So far we only have PHA and ZLK, if more assets exist, I will use a table